srcts/src/shiny/bind.ts

```diff
@@ -151,6 +169,7 @@ const bindingsRegistery = (() => {
   }
 
   return {
+    isRegistered,
```

Comment:
Probably shouldn't be exported to minimize the surface area of the registry object. See the below comment for more context.